### PR TITLE
Show both assigned and generated external id

### LINKED ISSUE #2164: Show both external id and rems id for an application
If an application has been assigned an external id, rems ui should show both the external id and the rems id for the application

### PR DESCRIPTION
**Pull request by Macroz**
Closes #2164 

I also decided to make the actions screen use the full width of the screen. The table can use the extra space especially with longer ids.

wide_external_id

## Reviewability
- [x] Link to issue
- [ ] Note if related change in rems-deploy repo (going to do this next)
- [x] Consider adding screenshots for ease of review

## Backwards compatibility
- [x] API is backwards compatible or completely new
- [x] Events are backwards compatible _or_ have migrations
- [x] Config is backwards compatible

## Documentation
- [x] Update changelog if necessary
- [x] API is documented and shows up in Swagger UI
- [x] New config options in config-defaults.edn

## Testing
- [x] Complex logic is unit tested
- [ ] Valuable features are integration / browser / acceptance tested automatically

## Follow-up
- [ ] New tasks are created for pending or remaining tasks

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ Changes since v2.15
 - The browser tests will now fail if there are any accessibility violations. (#2463)
 
 - The OIDC configuration is now validated, and REMS refuses to start without a valid OIDC configuration. See the `:oidc-metadata-url` configuration variable and [configuration.md](docs/configuration.md). (#2519)
+- The handler is shown both the assigned external id and the original REMS generated external id. These have also been added as new values to the data model and the original `:application/external-id` kept as it is. (#2614)
```
Comment (opqdonut) [inline CHANGELOG.md:23]: This sounds like the new behaviour is the default, but it's not, right?

File: src/clj/rems/db/applications.clj [hunk 2]
```diff
@@ -47,8 +47,10 @@
     (format-external-id (allocate-external-id! id-prefix))))
 
 (defn allocate-application-ids! [time]
-  {:application/id (:id (db/create-application!))
-   :application/external-id (application-external-id! time)})
+  (let [external-id (application-external-id! time)]
+    {:application/id (:id (db/create-application!))
+     :application/generated-external-id external-id
```
Comment (opqdonut) [inline src/clj/rems/db/applications.clj:52]: is this really used? application-base-view sets `:application/generated-external-id (:application/external-id event)`

**Review by opqdonut — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (Macroz) [reply CHANGELOG.md:23]: True, I'll change the wording a bit to clarify how it is taken into use.

File: src/clj/rems/db/applications.clj (hunk 2 above)
Comment (Macroz) [reply src/clj/rems/db/applications.clj:52]: I guess this got left over as a half solution. I'll remove this.

**Review by opqdonut — APPROVED** [review]

**Pull request merged**